cms/djangoapps/contentstore/rest_api/v1/tests/test_home.py

```diff
@@ -18,6 +19,7 @@ class HomePageViewTest(CourseTestCase):
     """
     Tests for HomePageView.
     """
+    MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
```

Comment:
This add new modulestore, the next commit add the import xD

Reply:
My mistake I made a interactive rebase a this line was moved from the other commit